src/main/src/main/java/org/geoserver/GeoserverInitStartupListener.java

```diff
@@ -220,6 +189,49 @@ private boolean isSerializableRenderedImageFinalization(Object where, Throwable
         Hints.putSystemDefault(Hints.EXECUTOR_SERVICE, executor);
     }
 
+    /** Initializes the default JAI ImagingListener */
+    public static void initJAIDefaultInstance() {
+        JAI jaiDef = JAI.getDefaultInstance();
+        // Custom ImagingListener used to ignore common warnings
+        if (!(jaiDef.getImagingListener() instanceof GeoServerImagingListener)) {
+            jaiDef.setImagingListener(new GeoServerImagingListener());
+        }
+    }
+
+    private static final class GeoServerImagingListener implements ImagingListener {
+        private static final Logger LOGGER = Logging.getLogger("org.geoserver.jai");
```

Comment:
I am not sure how I feel about changing the logger in this manner, if you wish to do so please note the change in the update instrutions also.

aside: Looking ahead to GeoServer 3 we will be changing to ImageN shortly, can you think of a different logger that would not need to get renamed?

Reply:
I'll just drop the logger change. The main focus of the PR should just be to throw the exceptions.